### DISCUSSION TITLE
Amazing tool

### DISCUSSION THREAD

**Discussion by Vanceral** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#164] [General]
Worked perfectly on Xiaomi 11 Lite 5G NE. 
Huge respect! 
Stumbled by Request parameter error (10000), but super easy fix on issue #82 of updating settings.apk. Thanks again.

**Comment by murdegern** [comment]
I have the same phone, but it didn't work for me :-( the unlock app still says "This device is not activated, please activate it and try to unlock it again."
